{{cookiecutter.repo_name}}/requirements/docs.txt

```diff
@@ -1,2 +1,2 @@
-Sphinx
+Sphinx==1.6.7 ## Pinned due to bug in 1.7.0 - 1.7.1
```

Comment:
Which bug?  If there's a GitHub issue for it, please add a link so we know when to unpin it again